apollo-adminservice/src/main/scripts/startup.sh

```diff
@@ -97,14 +97,14 @@ if [[ "$javaexe" ]]; then
     version=$(echo "$version" | awk -F. '{printf("%03d%03d",$1,$2);}')
     # now version is of format 009003 (9.3.x)
     if [ $version -ge 011000 ]; then
-        JAVA_OPTS="$JAVA_OPTS -Xlog:gc*:$LOG_DIR/gc.log:time,level,tags -Xlog:safepoint -Xlog:gc+heap=trace"
+        JAVA_OPTS="$JAVA_OPTS -Xlog:gc*:$LOG_DIR/$SERVICE_NAME-gc.log:time,level,tags -Xlog:safepoint -Xlog:gc+heap=trace"
```

Comment:
Is it better to have the name as `apollo-configservice.gc.log` versus `apollo-configservice-gc.log`?
```suggestion
        JAVA_OPTS="$JAVA_OPTS -Xlog:gc*:$LOG_DIR/$SERVICE_NAME.gc.log:time,level,tags -Xlog:safepoint -Xlog:gc+heap=trace"
```

Reply:
`apollo-configservice.gc.log` looks better